cmd/common.go

```diff
@@ -101,6 +101,13 @@ func syncMain(ctx context.Context, filenames []string, dry bool, parallelism,
 	var kongClient *kong.Client
 	mode := getMode(targetContent)
 	if mode == modeKonnect {
+		if targetContent.Workspace != "" {
+			return fmt.Errorf("_workspace entry found in config file.\n" +
+				"Workspaces are not supported in Konnect")
```

Comment:
Please make the error more actionable. What is a user supposed to do after this error?
Something along the lines (same applies for the other error):
```suggestion
			return fmt.Errorf("_workspace set in config file.\n" +
				"Workspaces are not supported in Konnect. Please remove '_workspace: foo' from your configuration and try again")
```